plugins/source/azure/client/client.go

```diff
@@ -43,9 +43,10 @@ func (c *Client) discoverSubscriptions(ctx context.Context) error {
 			return err
 		}
 		for _, sub := range page.Value {
+			// we record all returned values, even disabled
+			c.SubscriptionsObjects = append(c.SubscriptionsObjects, sub)
```

Comment:
Im not sure we want disabled projects as it will increase the load on APIs significantly and will slow the syncing.

Reply:
This object is used only in fetch. If this object is nil the fetch is still returning all values no matter what state the subscription is in.